<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BankWindow</class>
 <widget class="QMainWindow" name="BankWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bank</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #121214;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Home</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QWidget" name="widget_3" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QWidget" name="widget_5" native="true">
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <item>
               <widget class="QLabel" name="label_balance">
                <property name="font">
                 <font>
                  <pointsize>18</pointsize>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: rgb(255, 255, 255);</string>
                </property>
                <property name="text">
                 <string>Balance:</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget_4" native="true"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <pointsize>11</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>Options</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="pushButton_deposit">
              <property name="font">
               <font>
                <family>Nirmala UI</family>
                <pointsize>12</pointsize>
                <bold>false</bold>
               </font>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
	border:1px solid rgb(92,92,92);
}

QPushButton:hover{
	background-color: #191919;
}

</string>
              </property>
              <property name="text">
               <string>DEPOSIT</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_withdraw">
              <property name="font">
               <font>
                <family>Nirmala UI</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
	border:1px solid rgb(92,92,92);
}

QPushButton:hover{
	background-color: #191919;
}

</string>
              </property>
              <property name="text">
               <string>WITHDRAW</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string> history</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QWidget" name="widget_6" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>60</height>
           </size>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QWidget" name="widget_9" native="true">
             <widget class="QComboBox" name="comboBox_transaction_type">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>0</y>
                <width>181</width>
                <height>31</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <family>Nirmala UI</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="styleSheet">
               <string notr="true">color:  black;
background-color: rgb(255, 255, 255);
border-radius: 5px;</string>
              </property>
              <item>
               <property name="text">
                <string>Deposit</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Withdraw</string>
               </property>
              </item>
             </widget>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget_8" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLabel" name="label_quantity_found_history">
                <property name="styleSheet">
                 <string notr="true">color: rgb(255, 255, 255);</string>
                </property>
                <property name="text">
                 <string>Found:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_4">
                <property name="styleSheet">
                 <string notr="true">color: rgb(255, 255, 255);</string>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_7" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>3</verstretch>
           </sizepolicy>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="1" column="0">
            <widget class="QTableWidget" name="tableWidget_history">
             <property name="styleSheet">
              <string notr="true">
QTableWidget{
	border: none;
	color: white;

}

QTableWidget:item{
	color: white;
}</string>
             </property>
             <attribute name="verticalHeaderMinimumSectionSize">
              <number>20</number>
             </attribute>
             <column>
              <property name="text">
               <string>Date</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>amount</string>
              </property>
             </column>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="Line" name="line">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>830</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionLeave"/>
   </widget>
   <addaction name="menuOptions"/>
  </widget>
  <action name="actionLeave">
   <property name="text">
    <string>Leave</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
